From d3cf3fac7b6efe0800ed6cd912eb3b0ae2d78430 Mon Sep 17 00:00:00 2001
From: Zhao Chao <zhaochao1984@gmail.com>
Date: Wed, 26 Nov 2014 08:26:42 +0800
Subject: [PATCH 1/2] rwtab: Add /var/lib/ovirt-hosted-engine-ha.

/var/lib/ovirt-hosted-engine-ha/ha.conf need to be writable
when setting maintenance.

Signed-off-by: Zhao Chao <zhaochao1984@gmail.com>
---
 rwtab/hosted-engine | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rwtab/hosted-engine b/rwtab/hosted-engine
index 009d7f2..b1fff91 100644
--- a/rwtab/hosted-engine
+++ b/rwtab/hosted-engine
@@ -1,2 +1,3 @@
 # hosted-engine paths for read and write
 files /var/lib/ovirt-hosted-engine-setup
+files /var/lib/ovirt-hosted-engine-ha
-- 
1.8.3.1

